app/views/shared/_show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= medium.name %> <small><%= create_type %> by: <%= creator %></small></h1>
+<h4>Ranked: <%= medium.ranked %></h4>
+<p>
+  <%= medium.description %>
+</p>
+<%= button_to "Upvote", polymorphic_path([medium, :upvote]), method: :patch, class: "btn btn-primary" %>
+<%= link_to "Edit #{medium.name}", edit_polymorphic_path(name), method: :get, class: "btn btn-default" %>
+<%= link_to "DELETE", "/#{name.pluralize}/#{medium.id}", method: "delete",  data: { confirm: "Are you sure?" }, class: "btn btn-danger" %>
```

Comment:
This should use a route helper method like all of your other links/buttons. For the `destroy` action we can use the same helper we use for `show`, `polymorphic_path(medium)`. As long as we retain the `method: "delete"` part, Rails will be able to figure it out and do the right thing.